### PR TITLE
Monthly report for March 2024

### PR DESCRIPTION
**Pull request by ch1bo**
The same procedure as every month..

### CODE REVIEW

**Comment by ch1bo** [comment]
@rezabaram your mithril status update is still missing, but I'll mark this ready for review already so everyone can review my ramblings (especially conclusion)

**Comment by jpraynaud** [comment]
> @rezabaram your mithril status update is still missing, but I'll mark this ready for review already so everyone can review my ramblings (especially conclusion)

I have added the missing section for Mithril protocol status update @ch1bo :+1:

File: monthly/2024-03-monthly.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+After an initial design phase resulting in [ADR29](https://hydra.family/head-protocol/adr/29/), they contributed the event source and sink abstractions, and refactored the Hydra node to use these new extension points together with core contributors. The `EventSource` and `EventSink` interfaces, along with the configurable list of `eventSinks` in the main `HydraNode` handle, allow to extend the `hydra-node` with any external system. It also paves the way for more scalable persistence methods on the mainline Hydra node, and might even allow to refactor the existing API server as an `EventSink`.
+
+## Community
+
+### Mithril office hours
+
+As the catalyst project of TxPipe on Mithril is completed, we want to build upon this initial maintenance work in a more open format. For this, we are transforming our alignment calls into a public weekly office hour where we invite the community to join and discuss concrete pull requests, issues and questions on Mithril. 
+
+:point_right: Mitril office hours happen every Tuesday at 16:00 UTC on the [#mithril-live](https://discord.com/channels/826816523368005654/1146481948794032260) discord channel.
```
Comment (jpraynaud) [inline monthly/2024-03-monthly.md:123]:
```suggestion
:point_right: Mithril office hours happen every Tuesday at 16:00 UTC on the [#mithril-live](https://discord.com/channels/826816523368005654/1146481948794032260) discord channel.
```

**Review by jpraynaud — COMMENTED** [review]
LGTM :+1: 

Just noticed a typo

**Review by v0d1ch — APPROVED** [review]

**Comment by ch1bo** [comment]
@oduameh @olgahryniuk this includes now your changes from #4. I assume you approve?

**Pull request merged**